changelog.md

```diff
@@ -1,25 +1,60 @@
-# 4.0.6 / 2022-01-18
+# 4.1.0 / 2022-01-21
+
+### New Features
+
+- New kanban feature
+- Improvement of the responsive aspect of the interface
+``
+(responsive is an adjective)
+- News feed reversed
+- Calibration of the cover and avatar profile photos
+- Improvement of user's space management for the admin
+- Improvement of the call feature (bugfix and polish)
+- UI/UX improvement (buttons, redirections)
+- Performance improvement: better image and PDF preview generation, loading of space list and user list
+- Reactivation of the collabora draw functionality
 
 ### Fixed Issues
 
-- Backend: [#5253](https://github.com/tracim/tracim/issues/5253)
+- Frontend: [#4907](https://github.com/tracim/tracim/issues/4907),
+[#5136](https://github.com/tracim/tracim/issues/5136)
+- Backend: [#4483](https://github.com/tracim/tracim/issues/4483),
+[#4494](https://github.com/tracim/tracim/issues/4494),
+[#4648](https://github.com/tracim/tracim/issues/4648),
+[#4718](https://github.com/tracim/tracim/issues/4718),
+[#4955](https://github.com/tracim/tracim/issues/4955),
+[#4985](https://github.com/tracim/tracim/issues/4985),
+[#5008](https://github.com/tracim/tracim/issues/5008)
 
+### Breaking/Important change
 
-# 4.0.5 / 2021-12-29
+- Poppler-utils and webp now need to be present (and qpdf can be gotten rid off) on Tracim installations that don't use Docker (pr [#5121](https://github.com/tracim/tracim/pull/5121)),
+- By default, EPS previews are now disabled for security reasons, unless you manually enabled them. ([see preview generator documentation](https://github.com/algoo/preview-generator#eps-support))
+- Docker image support for ARM64 ([#3985](https://github.com/tracim/tracim/issues/3985))
+- Allow changing the parent of a space through tracimcli ([#3636](https://github.com/tracim/tracim/issues/3636))
+- Improve and add caldav (agenda) ([#2099](https://github.com/tracim/tracim/issues/2099)) and add carddav (adressbook) support ([#1469](https://github.com/tracim/tracim/issues/1469))
```

Comment:
I not seen link to the migration documentation. What I need to do to upgrade Tracim ?

Reply:
Is it correct if I add : 

- :warning: You need to migrate your database before running this version. For more information see: [#5118](https://github.com/tracim/tracim/issues/5118)

Reply:
"You need to adjust some settings before running this version"

(nothing actually related to the database needs to be done)

Reply:
:warning: You need to adjust some settings before running this version. For more information see: [#5118](https://github.com/tracim/tracim/issues/5118)